### PR TITLE
OAM-121: Added quarantined field to Lot entity

### PR DESCRIPTION
**Pull request by sradziszewski**
JIRA task link: https://openlmis.atlassian.net/browse/OAM-121

### CODE REVIEW

File: src/integration-test/java/org/openlmis/referencedata/web/LotControllerIntegrationTest.java [hunk 1]
```diff
@@ -113,6 +114,7 @@ public void shouldCreateNewLot() {
     assertEquals(lotDto.getLotCode(), response.getLotCode());
     assertEquals(lotDto.isActive(), response.isActive());
     assertEquals(lotDto.getTradeItemId(), response.getTradeItemId());
+    assertEquals(lotDto.isQuarantined(), response.isQuarantined());
```
Comment (tsznaj) [inline src/integration-test/java/org/openlmis/referencedata/web/LotControllerIntegrationTest.java:117]: To be concise, please add checks also in other methods as:
- shouldUpdateLot
- shouldGetLot

Comment (sradziszewski) [reply src/integration-test/java/org/openlmis/referencedata/web/LotControllerIntegrationTest.java:117]: I added extra checks.

Comment (tsznaj) [reply src/integration-test/java/org/openlmis/referencedata/web/LotControllerIntegrationTest.java:117]: Great!

**Review by tsznaj — APPROVED** [review]
Well done!

**Pull request merged**